src/sinks/aws_cloudwatch_logs.rs

```diff
@@ -58,35 +87,31 @@ enum State {
 pub enum CloudwatchError {
     Put(PutLogEventsError),
     Describe(DescribeLogStreamsError),
+    CreateStream(CreateLogStreamError),
     NoStreamsFound,
     ServiceDropped,
+    MakeService,
 }
 
 #[typetag::serde(name = "aws_cloudwatch_logs")]
 impl crate::topology::config::SinkConfig for CloudwatchLogsSinkConfig {
     fn build(&self, acker: Acker) -> Result<(super::RouterSink, super::Healthcheck), String> {
-        let cloudwatch = CloudwatchLogsSvc::new(self.clone())?;
-
-        let timeout = self.request_timeout_secs.unwrap_or(60);
-        let in_flight_limit = self.request_in_flight_limit.unwrap_or(5);
-        let rate_limit_duration = self.request_rate_limit_duration_secs.unwrap_or(1);
-        let rate_limit_num = self.request_rate_limit_num.unwrap_or(5);
+        let svc = CloudwatchLogsPartitionSvc::new(self.clone())?;
 
         let batch_timeout = self.batch_timeout.unwrap_or(1);
-        let batch_size = self.batch_size.unwrap_or(bytesize::mib(1u64) as usize);
+        let batch_size = self.batch_size.unwrap_or(1000);
```

Comment:
Nitpick, but might be nice to keep using `bytesize` here just for clarity and consistency.

Reply:
#574